### DISCUSSION TITLE
Are Next plugins needed after build for production?

### DISCUSSION THREAD

**Discussion by Akumzy** [vercel/next.js#13131] [Help] [answered]
I am currently automating my deployment process with GitHub Actions which I am stripping off all the DevDependencies but kept on having issues with Next plugins not installed.
So I want to know if those plugins are required after build

**Comment by jamesmosier** [comment, marked as answer]
To run your app via `next start` you'll need to also have your plugin dependencies used in your `next.config.js` installed.

**Reply by Akumzy** [reply]
Got it thanks
